cygnus-ngsi/resources/ngsi-examples/deprecated/ngsiv1-notification-json-geom.sh

```diff
@@ -37,31 +37,41 @@ else
    SERVICE_PATH=/
 fi
 
-curl $URL -v -s -S --header 'Content-Type: application/json; charset=utf-8' --header 'Accept: application/json' --header 'User-Agent: orion/0.10.0' --header "Fiware-Service: $SERVICE" --header "Fiware-ServicePath: $SERVICE_PATH" --header "ngsiv2-attrsformat: normalized" -d @- <<EOF
```

Comment:
What I have done is to move all the previous .sh to the deprecated/ directory (adding the "ngsiv1-" prefix to each file) and move the content in ngsiv2/ to the directory.

However, github interprets this as a change in the previous files.

NTC (informative)